Resources/Prototypes/Objectives/thief.yml

```
@@ -15,7 +15,6 @@
   id: BaseThiefStealObjective
   components:
   - type: StealCondition
```

```suggestion
  - type: StealCondition
    verifyMapExistence: false
```
Turns out this is needed after all, because some items are only produced during the round, for example tech disks or mail. But we can keep it for the animal objective